rosbag2_storage/include/rosbag2_storage/storage_interfaces/base_write_interface.hpp

```diff
@@ -38,6 +38,8 @@ class ROSBAG2_STORAGE_PUBLIC BaseWriteInterface
   virtual void create_topic(const TopicMetadata & topic) = 0;
 
   virtual void remove_topic(const TopicMetadata & topic) = 0;
+
+  virtual void split_database() = 0;
```

Comment:
I don't think this change should be on the interface as each individual storage shouldn't care about splitting itself.